src/ogr/geometry.jl

```diff
@@ -325,7 +325,13 @@ getgeomtype(geom::AbstractGeometry)::OGRwkbGeometryType = _geomtype(geom)
 
 Fetch WKT name for geometry type.
 """
-geomname(geom::AbstractGeometry)::String = GDAL.ogr_g_getgeometryname(geom.ptr)
+function geomname(geom::AbstractGeometry)::Union{String, Missing}
+    if geom.ptr == C_NULL
+        return missing
+    else
+        GDAL.ogr_g_getgeometryname(geom.ptr)
```

Comment:
For consistency a return statement here would be good as well. It's probably a good idea to run JuliaFormatter as described here: https://github.com/yeesian/ArchGDAL.jl#style-guide.

Reply:
Done, I have run the JuliaFormatter and applied @yeesian suggestion